app/api/resources/admin.py

```diff
@@ -76,3 +77,37 @@ def post(cls):
 
         else:
             return messages.USER_REVOKE_NOT_ADMIN, 403
+
+
+@admin_ns.route('admins')
+class ListAdmins(Resource):
+
+    @classmethod
+    @jwt_required
+    @admin_ns.doc('get_list_of_admins')
+    @admin_ns.doc(responses={
+        401: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
```

Comment:
can you also document error 403 you are returning in the end of this method?
here's an example: https://github.com/anitab-org/mentorship-backend/pull/459/files#diff-40b144b4232d9ee0bc35b84eb4226c7cR52

Reply:
I have now added docs for 200 and 403

Reply:
I have now added docs for 200 and 403